### PR TITLE
show todos for all instructors

### PR DESCRIPTION
**Pull request by jshawl**

### CODE REVIEW

File: app/views/cohorts/todos.html.erb [hunk 1]
```diff
@@ -0,0 +1,26 @@
+<h2>Number Assignments ungraded</h2>
+<div class='chart todos-chart'></div>
+<script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/d3/3.5.16/d3.min.js"></script>
```
Comment (adambray) [inline app/views/cohorts/todos.html.erb]: should we vendor this?

File: app/views/cohorts/todos.html.erb [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+<script>
```
Comment (adambray) [inline app/views/cohorts/todos.html.erb]: Can this be moved into a js file?

File: app/assets/javascripts/todos.js.erb [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+    .data(todoData)
+    .enter()
+    .append("div")
+    .style("width", function(d){
+      return scale(d.ungraded) + "%"
+    })
+    .text(function(d){
+      return d.user + ": " + d.ungraded
+    })
+}
```
Comment (RobertAKARobin) [inline app/assets/javascripts/todos.js.erb]: Does this need to be an ERB?

**Comment by RobertAKARobin** [comment]
:+1: pending tests!

**Pull request merged**